__tests__/authentication.test.js

```diff
@@ -45,40 +44,50 @@ describe('authentication', () => {
       .onPost(`${baseURL}/users/authenticate`, { email: 'test@alenvi.io', password: '1234567' })
       .reply(
         200,
-        { data: { token: 'token', tokenExpireDate: '1234567890', refreshToken: 'refreshToken', userId: 'userId' } }
+        {
+          data: {
+            token: 'token',
+            tokenExpireDate: '1234567890',
+            refreshToken: 'refreshToken',
+            user: { _id: 'userId' },
+          },
+        }
       );
 
     const currentDate = new Date();
-    const date = new Date(currentDate.getFullYear(), currentDate.getMonth(), currentDate.getDate());
-    loggedAxiosMock.onGet(`${baseURL}/users/usersId`, { params: { mobileVersion: '1.0.0', appName: 'erp' } })
-      .reply(200, { data: { _id: 'userId' } })
+    const startDate = new Date(currentDate.getFullYear(), currentDate.getMonth(), currentDate.getDate(), 0, 0, 0, 0);
+    const endDate = new Date(currentDate.getFullYear(), currentDate.getMonth(), currentDate.getDate(), 23, 59, 59, 999);
+
+    loggedAxiosMock.onGet(`${baseURL}/users/userId`)
+      .reply(200, { data: { user: { _id: 'userId' } } })
       .onGet(
         `${baseURL}/events`,
-        { params: { auxiliary: 'userId', startDate: date, endDate: date, type: INTERVENTION } }
+        { params: { auxiliary: 'userId', startDate, endDate, type: INTERVENTION, isCancelled: false } }
       )
-      .reply(200, { data: [] });
+      .reply(200, { data: { events: [{ _id: 'eventId', startDate: new Date() }] } });
 
     const element = render(
       <AuthProvider>
         <AppContainer />
       </AuthProvider>
     );
 
-    const emailInput = await element.findByTestId('Email');
```

Comment:
`findByTestId` lance le warning : `You called act(async () => ...) without await` mais je ne comprends pas pourquoi. Et puisque `getByTestId` était utilisé dans Formation, j'ai préféré utiliser cette fonction.